Plugins/Statistics/GitStatistics/GitStatisticsPlugin.cs

```diff
@@ -10,7 +10,9 @@ public class GitStatisticsPlugin : GitPluginBase, IGitPluginForRepository
         StringSetting CodeFiles = new StringSetting("Code files",
                                 "*.c;*.cpp;*.cc;*.h;*.hpp;*.inl;*.idl;*.asm;*.inc;*.cs;*.xsd;*.wsdl;*.xml;*.htm;*.html;*.css;" + 
                                 "*.vbs;*.vb;*.sql;*.aspx;*.asp;*.php;*.nav;*.pas;*.py;*.rb;*.js");
-        StringSetting IgnoreDirectories = new StringSetting("Directories to ignore (EndsWith)", "\\Debug;\\Release;\\obj;\\bin;\\lib");
+
+        StringSetting IgnoreDirectories = new StringSetting("Ignore directories containing the following names", "\\Debug;\\Release;\\obj;\\bin;\\lib");
+        BoolSetting UseEndsWith = new BoolSetting("Use old directory filtering (EndsWith)", false);
```

Comment:
Why don't use a regex and get rid of the UseEndsWith? One could use `$` to match end of directory name.